### ISSUE TITLE
Barlow Condensed: Extra space after character ø

### ISSUE THREAD

**Issue by timclausen** [google/fonts#1851]
I'm having issues with Barlow Condensed when using it in Danish. Letter ø gets an additional spacing.

<img width="1168" alt="skaermbillede 2019-02-19 kl 14 05 42" src="https://user-images.githubusercontent.com/2226613/53017145-9ba73c80-344f-11e9-958a-6c79f884d5b5.png">

**Comment by m4rc1e** [comment]
Confirmed 

<img width="343" alt="screenshot 2019-02-19 at 13 11 54" src="https://user-images.githubusercontent.com/7525512/53017503-094f6a80-3448-11e9-8f2e-15f3a12465eb.png">

I'll check the source now.

**Comment by Emetico** [comment]
This is still not fixed when the word starts with a big Ø
![Udklip](https://user-images.githubusercontent.com/10114290/66381832-57ef4c00-e9ba-11e9-8359-18750a721bb1.PNG)

**Comment by fitojb** [comment]
@carnevalle The font’s source was updated, but the change still hasn’t reached the Google Fonts CDN. Please test with an updated build from https://github.com/jpt/barlow/tree/1.5/fonts as that version has a number of spacing and kerning improvements.

**Comment by RosaWagner** [comment]
I reopened this issue since the font still has be be upgraded on GF

**Comment by emmamarichal** [comment]
We are working on it (cc @RosaWagner), but the font file is quite complex (because of intermediate layers, filters, etc.), and it's difficult to build the font with fontmake.

**Comment by RosaWagner** [comment]
The last update is still in a branch and there is not a release yet, I suggest we let @jpt work on it. Please @jpt ping us when the upgrade is ready :)

**Comment by jpt** [comment]
indeed, I am actively working on a large number of issues in barlow, including the ø - although I changed it years back at this point to not substitute the alternate shape at heavier weights because a number of norwegian readers complained initially and who was I to disagree 😅 but if that kind of design change is controversial given the google cdn's reach, happy to talk about it.

in case it's helpful for anyone following barlow's development more generally, I recently updated some milestones here - https://github.com/jpt/barlow/milestones
the dates are possibly overly optimistic but I needed to put some goalposts somewhere :)

**Comment by carnevalle** [comment]
@jpt @RosaWagner 

Bad news. The Ø fix found in https://github.com/jpt/barlow/archive/refs/heads/1.5.zip has introduced a bug with percentage sign %. See attached screenshot.

![barlow](https://user-images.githubusercontent.com/54747/148049720-fa049a14-2fa4-41db-9fd4-de2b6ab99047.png)

**Comment by jpt** [comment]
Hi @carnevalle, the 1.5 branch is under active development and has more significant bugs than just the percentage sign right now; I wouldn't recommend using it until it's merged into the main branch. If you notice any other bugs, feel free to open an issue in the [Barlow repo](https://github.com/jpt/barlow/issues) along with the others so I can track it.

**Comment by carnevalle** [comment]
@jpt 

I'll do that. Do you want me to create an issue with this bug or is it already created somewhere?

**Comment by jpt** [comment]
oh, no, sorry I wasn;t more clear — I mean if you notice anything beyond the percentage sign, that one is done (though it's not yet in the font binaries it should be fixed in the Glyphs file)

**Comment by carnevalle** [comment]
@RosaWagner I can see these fixes are being moved from milestone to milestone. Is there any news about when a release would be expected?

**Comment by jpt** [comment]
This should be fixed upstream in the 1.5 branch. I'm [drafting a PR](https://github.com/jpt/barlow/pull/99). Most of what I want to do before merging it (and opening a separate PR here) is to check some things in fontbakery and also double check some things related to the vertical metrics.
